Carthage.xcodeproj/project.pbxproj

```diff
@@ -1091,7 +1091,7 @@
 				CLANG_WARN_STRICT_PROTOTYPES = YES;
 				CLANG_WARN_SUSPICIOUS_MOVE = YES;
 				GCC_NO_COMMON_BLOCKS = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.11;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
don't we have a dependency on Xcode 9.4? This requires macOS 10.13.2+

Reply:
Yes, but this is the _deployment_ target. You need macOS 10.13.2+ to build, but you can use the built binary on older platforms.

SwiftPM doesn't currently have a way to specify a deployment target and defaults to 10.10. This just aligns the Xcode project with the SwiftPM default.